src/workflows/airqo_etl_utils/airqo_api.py

```diff
@@ -104,7 +104,7 @@ def get_devices(
         tenant: Tenant = Tenant.ALL,
         device_category: DeviceCategory = DeviceCategory.NONE,
     ) -> list:
-        params = {"tenant": str(Tenant.AIRQO)}
+        params = {"tenant": str(Tenant.AIRQO), "active": "yes"}
```

Comment:
thanks @MartinKalema ....in future PRs, we shall also need to revisit the TENANT part since we no longer used this concept in our respective microservices.